tests/suites/test_suite_psa_crypto.function

```diff
@@ -712,9 +712,9 @@ exit:
     return( ok );
 }
 
-static int asn1_skip_integer( unsigned char **p, const unsigned char *end,
```

Comment:
This change popped up for me during reproduction of the merge. It doesn't seem to be used elsewhere, hence could be left static.

Reply:
It's non-static because it's unused in some build configurations, and the compiler complains that a function is unused. It didn't use to be a problem because we used to compile tests with `-Wno-unused`, but since this merge, we've stopped doing that. This is explained in the merge commit message for `test_suite_psa_crypto_slot_management.function`. I didn't list it for `test_suite_psa_crypto.function` because there it's a change that already happened in the development branch, there was no conflict.